linkerd/tls/src/server/mod.rs

```diff
@@ -236,25 +245,8 @@ where
     while io.read_buf(&mut buf).await? != 0 {
         debug!(buf.len = %buf.len(), "Read bytes from TCP stream");
         match client_hello::parse_sni(buf.as_ref()) {
-            Ok(Some(ServerId(sni))) if sni == local_id => {
-                trace!(%sni, "Identified matching SNI via buffered read");
```

Comment:
nit, not important: looks like we no longer log the SNI (or whether it matches or not); is this still potentially worth having?